WISE-Community/WISE-Client
# feat(Export): Implement Label component export

## Changes

Implemented Label component export.

## Test

Make sure the Label component export shows the labels the students saved.
1. Open the Classroom Monitor
2. Go to the "Data Export" view
3. Click "Export Component Data"
4. Find a Label component and click "All" and "Latest"

Also make sure changing the settings for "Include Student Names" and "Include Only Submits" are properly reflected in the export file.

Closes #923
